jmclient/jmclient/wallet_utils.py

```diff
@@ -43,26 +44,20 @@ def get_wallettool_parser():
                       dest='showprivkey',
                       help='print private key along with address, default false')
     parser.add_option('-m',
-                      '--maxmixdepth',
+                      '--mixdepth',
                       action='store',
                       type='int',
-                      dest='mixdepths',
-                      help='how many mixing depths to initialize in the wallet',
-                      default=5)
+                      dest='mixdepth',
+                      help="Mixdepth to use in the wallet. Default: {}"
+                           .format(DEFAULT_MIXDEPTH),
+                      default=None)
```

Comment:
Why is the default `None` and not `DEFAULT_MIXDEPTH`?

Reply:
The default value `None` shows all initialised mixdepths in the wallet for display[all]. The default value of `DEFAULT_MIXDEPTH` here is only used when creating a new wallet. This reminds me, for `importprivkey` there needs to be a check that this in not `None`.

Reply:
Thanks